comit/src/expiries/config.rs

```diff
@@ -34,16 +34,25 @@ use time::Duration;
 // to turn off lints.
 
 mod main {
-    pub const BITCOIN_BLOCK_TIME_SECS: u16 = 600; // 10 minutes, average Bitcoin block time.
-    pub const ETHEREUM_BLOCK_TIME_SECS: u16 = 20; // Conservative Ethereum block
-                                                  // time.
-
-    pub const BITCOIN_CONFIRMATIONS: u8 = 6; // Standard in the Bitcoin ecosystem.
-    pub const ETHEREUM_CONFIRMATIONS: u8 = 30; // Value used by Kraken.
-
-    pub const ACT_IN_SOFTWARE_SECS: u32 = 15 * 60; // Value arbitrarily chosen.
-    pub const ACT_WITH_USER_INTERACTION_SECS: u32 = 60 * 60; // Value arbitrarily
-                                                             // chosen.
+    use super::*;
+    use conquer_once::Lazy;
+    use time::NumericalDuration;
+
+    // 10 minutes, average Bitcoin block time.
+    pub static BITCOIN_BLOCK_TIME: Lazy<Duration> = Lazy::new(|| 600.seconds());
```

Comment:
`Duration` has `const` constructors that you can use: https://docs.rs/time/0.2.22/time/struct.Duration.html#method.seconds

Reply:
Oh, good tip. One `const` still has to go away tho.